saleor/tests/e2e/vouchers/test_export_voucher_codes.py

```diff
@@ -0,0 +1,496 @@
+                },
+            ],
+        ),
+        (
+            "CSV",
+            "invalid_voucher_id",
+            [
+                {
+                    "message": "Invalid voucher ID.",
+                    "code": "INVALID",
+                    "field": "voucherId",
+                },
+            ],
+        ),
+    ],
+)
+def test_export_voucher_codes_with_invalid_voucher_id_CORE_0925(
+    e2e_staff_api_client,
+    permission_manage_channels,
+    permission_manage_shipping,
+    permission_manage_products,
+    permission_manage_discounts,
+    permission_manage_checkouts,
+    file_type,
+    invalid_voucher_id,
+    expected_errors,
+):
+    # Before
+    permissions = [
+        permission_manage_channels,
+        permission_manage_shipping,
+        permission_manage_products,
+        permission_manage_discounts,
+        permission_manage_checkouts,
+    ]
+    assign_permissions(e2e_staff_api_client, permissions)
+
+    # Step 1 - Export voucher codes and check status
+    input_data = {
+        "fileType": file_type,
+        "voucherId": invalid_voucher_id,
+    }
+    response = raw_export_voucher_codes(e2e_staff_api_client, input_data)
+
+    assert response["data"]["exportVoucherCodes"]["errors"] == expected_errors
+
+
+@pytest.mark.e2e
+@pytest.mark.parametrize(
+    ("file_type", "invalid_voucher_code", "expected_errors"),
+    [
+        (
+            "XLSX",
+            "invalid_voucher_code",
+            [
+                {
+                    "message": "Invalid voucher code IDs.",
+                    "code": "INVALID",
+                    "field": "ids",
+                },
+            ],
+        ),
+        (
+            "CSV",
+            "invalid_voucher_code",
+            [
+                {
+                    "message": "Invalid voucher code IDs.",
+                    "code": "INVALID",
+                    "field": "ids",
+                },
+            ],
+        ),
+    ],
+)
+def test_export_voucher_codes_with_invalid_voucher_codes_CORE_0925(
+    e2e_staff_api_client,
+    permission_manage_channels,
+    permission_manage_shipping,
+    permission_manage_products,
+    permission_manage_discounts,
+    permission_manage_checkouts,
+    file_type,
+    invalid_voucher_code,
+    expected_errors,
+):
+    # Before
+    permissions = [
+        permission_manage_channels,
+        permission_manage_shipping,
+        permission_manage_products,
+        permission_manage_discounts,
+        permission_manage_checkouts,
+    ]
+    assign_permissions(e2e_staff_api_client, permissions)
+
+    # Step 1 - Export voucher codes and check status
+    input_data = {
+        "fileType": file_type,
+        "ids": [invalid_voucher_code],
+    }
+    response = raw_export_voucher_codes(e2e_staff_api_client, input_data)
+
+    assert response["data"]["exportVoucherCodes"]["errors"] == expected_errors
+
+
+@pytest.mark.e2e
+@pytest.mark.parametrize(
```

Comment:
Both tests have the same error. There is no reason to store errors in parametrize.